packages/exporter-trace-otlp-grpc/src/util.ts

```diff
@@ -50,17 +51,21 @@ export function onInit<ExportItem, ServiceRequest>(
     .then(packageDefinition => {
       const packageObject: any = grpc.loadPackageDefinition(packageDefinition);
 
+      const selectedCompressionAlgo = { 'grpc.default_compression_algorithm': collector.compression };
+
       if (collector.getServiceClientType() === ServiceClientType.SPANS) {
         collector.serviceClient =
           new packageObject.opentelemetry.proto.collector.trace.v1.TraceService(
             collector.url,
             credentials,
+            selectedCompressionAlgo,
```

Comment:
IIUC, this parameter is named `options` in the gRPC client. i.e. it's not limited to compression algorithms. 
```suggestion
            options,
```